### ISSUE TITLE
Set focus on long flashlist

### ISSUE THREAD

**Issue by saimirg** [flexn-io/create#190]
Hi, I have along carouse/flashlist and i want to set the focus on a specific item when it shows up. It only works when the item is within 20items difference from the first one, if it is further it doesnt work.
How do i set the focus on a remote item ?

**Comment by aurimasmi** [comment]
Can you tell what property/method you are using to set focus on any non first item?

**Comment by saimirg** [comment]
I tried the focusOptions.hasPreferredFocus

**Comment by aurimasmi** [comment]
Right. `hasPreferredFocus` only on visible items and recycler has only certain amount of items visible initially. This is valid functionality what you asking I will add it to mine backlog and will do it

**Comment by saimirg** [comment]
hi @aurimasmi is there any workaround to it ? Is there any to fix it in the near future ?

**Comment by saimirg** [comment]
hi @aurimasmi , i'm using **initialScrollIndex** and it seem to be doing the work, but only when you scroll right afterward. If you scroll left, it kind of repeats its self. It takes you back to the initial scroll index. Is there any hack to this ? This is the implementation:

```
<FlashList
                            data={channels}
                            renderItem={rowRenderer}
                            horizontal
                            type="row"
                            estimatedItemSize={Dimensions.get("window").width / 1}
                            initialScrollIndex={50}
                        />
```
